osu.Game.Rulesets.Osu/Mods/OsuModRelax.cs

```diff
@@ -20,6 +20,7 @@ public class OsuModRelax : ModRelax, IUpdatableByPlayfield, IApplicableToDrawabl
     {
         public override string Description => @"You don't need to click. Give your clicking/tapping fingers a break from the heat of things.";
         public override Type[] IncompatibleMods => base.IncompatibleMods.Append(typeof(OsuModAutopilot)).ToArray();
+        public override double ScoreMultiplier => 0.8;
```

Comment:
@smoogipoo @peppy Should relax have score multiplier and would that even work?

Reply:
From what I've seen around, I am fairly assuming Relax will exist on the same leaderboards as non-Relax. For that reason, the score multiplier should be reduced to make up for it (it's currently 1.0x in master branch).

Reply:
Score multipliers are out of discussion currently and should not be changed.

Reply:
Alright, I'll remove it.